2-structured-data/package.json

```diff
@@ -54,7 +54,7 @@
     ]
   },
   "dependencies": {
-    "@google-cloud/datastore": "1.3.4",
+    "@google-cloud/datastore": "1.4.2",
```

Comment:
Let's not bundle this change in with the others.  We are using renovate to keep these up to date, and would like to keep small single-purpose commits for package version updates.  It makes figuring out breaks much, much easier :)

Reply:
+1

Reply:
Actually it's required; otherwise the tests will fail because it cannot find Cloud Datastore 1.3.4 and Cloud Pub/Sub 0.16.1 for Node.js 10